<?xml version="1.0" encoding="UTF-8"?><magicdraw><comment classType="Comment" humanType="Comment" icon="index_files/icon_comment_841774062.jpg" id="_10_5_622020d_1132928308069_11031_5133" refElementId="_10_5_622020d_1132928308069_11031_5133"><annotatedElement humanName="Annotated Element" mode="s,e"><class classType="Class" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="RuntimeException" refElementId="_9_5_f720368_1110473546612_799782_7673" refid="_9_5_f720368_1110473546612_799782_7673"/></annotatedElement><body humanName="Body" mode="s,e">&lt;code&gt;RuntimeException&lt;/code&gt; is the superclass of those
exceptions that can be thrown during the normal operation of the
Java Virtual Machine.
&lt;p&gt;
A method is not required to declare in its &lt;code&gt;throws&lt;/code&gt;
clause any subclasses of &lt;code&gt;RuntimeException&lt;/code&gt; that might
be thrown during the execution of the method but not caught.


@author  Frank Yellin
@version 1.13, 12/19/03
@since   JDK1.0
</body><documentation humanName="Documentation"/><owner classType="Class" humanName="Owner" humanType="Class" icon="index_files/icon_class_555864569.jpg" mode="s,e" name="RuntimeException" refElementId="_9_5_f720368_1110473546612_799782_7673" refid="_9_5_f720368_1110473546612_799782_7673"/><owningElement classType="Class" humanName="Owning Element" humanType="Class" icon="index_files/icon_class_555864569.jpg" name="RuntimeException" refElementId="_9_5_f720368_1110473546612_799782_7673" refid="_9_5_f720368_1110473546612_799782_7673"/><TO_DO humanName="To Do" mode="s,e"/></comment></magicdraw>